extensions/import/src/wizard/pages/prosePreviewPage.ts

```diff
@@ -95,8 +119,9 @@ export class ProsePreviewPage extends ImportPage {
 			{
 				component: this.table,
 				title: '',
-				actions: [this.refresh]
+				// actions: [this.refresh]
```

Comment:
Why are you removing this?

Reply:
The button serves no purpose so Aasim recommended I remove it. The code for the button will be fully removed next commit.